### PR TITLE
Update of the French translation

### PR DESCRIPTION
**Pull request by jeau**
@kevinpapst 
Here are the updated files for the French translation.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] I verified that my code applies to the guidelines (`composer kimai:code-check`)
- [ ] I updated the documentation (see [here](https://github.com/kimai/www.kimai.org/tree/master/_documentation))
- [X] I agree that this code is used in Kimai and will be published under the [MIT license](https://github.com/kevinpapst/kimai2/blob/master/LICENSE)

### CODE REVIEW

File: translations/actions.fr.xliff [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="fr" datatype="plaintext" original="none">
```
[the remark this answers is not in the document]
Comment (jeau) [reply translations/actions.fr.xliff]: I forgot that stuff. 
done for this file and the other two.

File: translations/daterangepicker.fr.xliff [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            </trans-unit>
+            <trans-unit id="daterangepicker.lastYear">
+                <source>daterangepicker.lastYear</source>
+                <target>L'an dernier</target>
+            </trans-unit>
+            <trans-unit id="daterangepicker.thisYear">
+                <source>daterangepicker.thisYear</source>
+                <target>Cette année</target>
+            </trans-unit>
+            <trans-unit id="daterangepicker.customRange">
```
Comment (kevinpapst) [inline translations/daterangepicker.fr.xliff]: This needs to be merged, its duplicated (see below)

Comment (jeau) [reply translations/daterangepicker.fr.xliff]: Sorry! I just deleted the duplicate ID.

**Comment by kevinpapst** [comment]
> Edit: and while you are on it, please merge with master to solve the conflict.

@jeau looking at your profile I assume you know what "merge master to solve the conflict" means. 
If not: I can do the merge as well, no problem: I just have to know ... otherwise I will wait, because I can't merge currently due to the conflicting file.

**Comment by kevinpapst** [comment]
Awesome, thanks @jeau !

**Pull request merged**